packages/Modal/src/Modal.tsx

```diff
@@ -166,88 +160,90 @@ const Modal: React.FunctionComponent<ModalProps> = (props: ModalProps) => {
       </AnimatePresence>
       <AnimatePresence onExitComplete={onExitComplete}>
         {isOpen && (
-          <m.div
-            key={`${keyValue}_modal`}
-            className={classes}
-            data-test={dataTest}
-            {...(overlayProps as any)}
-            {...dialogProps}
-            initial="initial"
-            animate="open"
-            exit="close"
-            variants={modalVariants}
-            transition={{ duration: 0.2 }}
-            ref={ref}
-          >
-            <div
-              className={cx('ids-modal__header', {
-                'ids-modal__header--with-back-btn': displayBackBtn,
-              })}
+          <m.div className="ids-modal-wrapper">
```

Comment:
I think it should follow the way we name the others like ids-modal__header so it should be:
```suggestion
          <m.div className="ids-modal__wrapper">
```

Reply:
I'm not sure in that case, because `__something` is more when an element is child of, but in this case, wrapper is not the child of the modal